src/common/transformations/src/transformations/common_optimizations/change_placeholder_types.cpp

```diff
@@ -18,11 +18,11 @@ using namespace ov;
 
 namespace {
 bool is_node_casts_to_float_or_shapeof(const Node* node) {
-    if (dynamic_cast<const opset8::ShapeOf*>(node)) {
+    if (dynamic_cast<const ov::opset8::ShapeOf*>(node)) {
         return true;
     }
-    auto convert = dynamic_cast<const opset8::Convert*>(node);
-    if (convert && convert->get_destination_type() == element::f32) {
+    auto convert = dynamic_cast<const ov::opset8::Convert*>(node);
+    if (convert && convert->get_destination_type() == ngraph::element::f32) {
```

Comment:
Do not use `ngraph` namespace, use `ov` instead

Reply:
done